tests/suite/fixtures.py

```diff
@@ -269,6 +273,9 @@ def crd_ingress_controller(cli_arguments, kube_apis, ingress_controller_prerequi
     print("------------------------- Create IC -----------------------------------")
     name = create_ingress_controller(kube_apis.v1, kube_apis.extensions_v1_beta1, cli_arguments, namespace,
                                      request.param.get('extra_args', None))
+    ensure_connection_to_public_endpoint(ingress_controller_endpoint.public_ip,
```

Comment:
will it make sense to add ` ensure_connection_to_public_endpoind` to `ingress_controller`?

Reply:
there are no flaky tests among 'ingress' tests, so I don't see any reason for that.